.github/workflows/benchmark.yml

```diff
@@ -56,9 +56,14 @@ jobs:
         with:
           workspaces: py-polars
 
+      - name: Use sccache
+        uses: mozilla-actions/sccache-action@v0.0.3
```

Comment:
Not sure how this works, but does this mean we can get rid of the `Swatinem/rust-cache` action?

Reply:
Are you fine with dropping `rust-cache`?

Reply:
I've left it in for now because I'm not sure what exactly it does.

Reply:
We should drop it if it's not adding anything. Cache space is limited. Running multiple caching solutions side-by-side is not an option, in my opinion.

You should post some benchmarks on the various workflows to show the new caching solution is an improvement over the existing method. Average runtime uncached vs cached. Maybe cache  space used. If it's a clear improvement then we can go ahead and merge this.

Reply:
The problem with benchmarks is that runtime on GHA varies tremendously. I’ve implemented Ccache caching in many repos and these kinds of benchmarks are impossible to do.